resources/healthlake-datastore.go

```diff
@@ -0,0 +1,63 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/healthlake"
+)
+
+type HealthlakeDatastore struct {
+        svc  *healthlake.HealthLake
+        datastoreId *string 
+}
+
+func init() {
+	register("HealthlakeDatastore", ListHealthlakeDatastores)
+}
+
+func ListHealthlakeDatastores(sess *session.Session) ([]Resource, error) {
+	svc := healthlake.New(sess)
+
+	resources := make([]Resource, 0)
+        params := &healthlake.ListFHIRDatastoresInput{}
+
+        for {
+                output, err := svc.ListFHIRDatastores(params)
+                if err != nil {
+                        return nil, err
+                }
+
+                for _, datastore := range output.DatastorePropertiesList {
+                        if *datastore.DatastoreStatus != "DELETED" {
```

Comment:
```suggestion
                        if *datastore.DatastoreStatus != healthlake.DatastoreStatusDeleted {
```
Another option here is to use the constant defined in the package [here](https://pkg.go.dev/github.com/aws/aws-sdk-go@v1.44.7/service/healthlake#pkg-constants)